Scripts/test.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+set -e
+
+function usage {
+    echo
+    echo "[!] Usage: $0 --platform [ios | tvos]"
+    echo
+    exit 1
+}
+
+if [[ "$1" != "--platform" ]]; then
+    usage
+fi
+
+PLATFORM="$2"
+if [[ $PLATFORM != "ios" && $PLATFORM != "tvos" ]]; then
+    usage
+fi
+
+Scripts/test-streams.sh -s
```

Comment:
I would maybe keep test stream management and test execution separate (only bundled together in the makefile), only combining them at the Makefile level.